### PR TITLE
 On branch update-security-md

### PR DESCRIPTION
**Pull request by lymtic**
Changes to be committed:
	modified:   content/zh/docs/concepts/security/index.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/concepts/security/index.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Istio Security 尝试提供全面的安全解决方案来解决所有这些问
 Istio 安全功能提供了强大的身份、强大的策略、透明的 TLS 加密、
 认证/授权/审计（AAA）工具来保护您的服务和数据。Istio 安全的目标是：
 
-- 默认安全：应用程序代码和基础设施无需更改
-- 深度防御：与现有安全系统集成以提供多层防御
-- 零信任网络：在不受信任的网络上构建安全解决方案
+- 默认安全：应用程序代码和基础设施无需更改；
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:36]: 英文也是句点，为什么要改成 `；`呢？有什么说法吗？
```
- To defend against man-in-the-middle attacks, they need traffic encryption.
```

**Review by windsonsea — COMMENTED** [review]
通常只需改不对的、不好的、不漂亮的，从 follow en 角度看，感觉 `；` 并不比 `。`好。

而且这样一改，以后一致性会是个大问题，因为正常人没有这样的习惯吧。

File: content/zh/docs/concepts/security/index.md [hunk 2]
```diff
@@ -377,9 +377,9 @@ spec:
     port: 8000
     protocol: TCP
     targetPort: 80
-  selector:
+    selector:
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:380]: You need to modify the en upstream for this kind of code firstly.

File: content/zh/docs/concepts/security/index.md [hunk 3]
```diff
@@ -377,9 +377,9 @@ spec:
...
     app: example-app
-{{< /text >}}
+    {{< /text >}}
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:382]:
```suggestion
{{< /text >}}
```
This is a incorrect change.

File: content/zh/docs/concepts/security/index.md [hunk 4]
```diff
@@ -525,7 +525,7 @@ spec:
  - from:
    - source:
        notNamespaces: ["foo"]
-{{< /text >}}
+       {{< /text >}}
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:528]:
```suggestion
{{< /text >}}
```
Same as above, this code change is incorrect.

**Review by windsonsea — CHANGES_REQUESTED** [review]
Pls keep the code exactly same as en upstream.

File: content/zh/docs/concepts/security/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/concepts/security/index.md]: 能举个权威的参考吗，`正确`需要有 reference。

File: content/zh/docs/concepts/security/index.md [hunk 5]
```diff
@@ -1,850 +1,852 @@
...
+
+  控制面处理来自 API server 的配置，并且在数据面中配置 PEP。
+  PEP 用 Envoy 实现。下图显示了架构。
+
+  {{< image width="75%"
+      link="./arch-sec.svg"
+      caption="安全架构"
+      >}}
+
+  在下面的部分中，我们将详细介绍 Istio 安全功能。
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:69]: Totally wrong! Why adding 2 spaces to each line?

**Review by windsonsea — CHANGES_REQUESTED** [review]
你这是运行脚本给每一行增加了 2 个空格？Why?

**Comment by windsonsea** [comment]
As a new contributor, a small PR might be more suitable. Don't change too much, which caused too many errors now.

**Comment by wilsonwu** [comment]
Suggest add prefix of the PR title like [zh], it could help reviewers make quick response.

File: content/zh/docs/concepts/security/index.md (hunk 5 above)
Comment (lymtic) [reply content/zh/docs/concepts/security/index.md]: sorry,md tool automatic formatting

**Comment by windsonsea** [comment]
Note the lint error:

```
./zh/docs/concepts/security/index.md:849: MD047 File should end with a single newline character
```

The blank line at bottom is required for a markdown file. Check the [markdownlint spec](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).

File: content/zh/docs/concepts/security/index.md [hunk 6]
```diff
@@ -848,3 +848,4 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
 
 - 阅读[常见问题](/zh/docs/ops/common-problems/security-issues/)，
   以便在出现问题时更好地解决安全策略问题。
+
```
Comment (wilsonwu) [inline content/zh/docs/concepts/security/index.md:851]:
```suggestion

```

Comment (wilsonwu) [reply content/zh/docs/concepts/security/index.md:851]: Some problem for the new line, forget this suggestion, just remove two spaces and leave the new line here is OK.

**Pull request closed without merging**